ext/civiimport/civiimport.php

```diff
@@ -275,6 +275,14 @@ function civiimport_civicrm_buildForm(string $formName, $form) {
   }
 
   if ($formName === 'CRM_Contact_Import_Form_Summary') {
+    $userJobID = $form->getUserJobID();
+    Civi::service('angularjs.loader')
+      ->addModules('afsearchImportSummaryImport' . $userJobID);
+    CRM_Core_Resources::singleton()->addMarkup('<crm-angular-js modules=\'afsearchImportSummaryImport' . $userJobID . '\'>
```

Comment:
I only converted these to single-quotes because I had converted the outer quotes to double. Just a personal preference, I like placing variable directly into double-quotes instead of concatenating them with single-quotes. I see you prefer the latter, so we can switch these back.

```suggestion
    CRM_Core_Resources::singleton()->addMarkup('<crm-angular-js modules="afsearchImportSummaryImport' . $userJobID . '">
```

Reply:
@colemanw the problem was the quotes were broken after I merged yours

Reply:
Yea it was a long string and I didn't do all of it.